### PR TITLE
Cleanup tags.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of animals._

jackalope

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/JPXzGji3

## Description
_The commit messages say what you did; this should explain why and how._
- [x] Description is in Trello card

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._

## Impacted areas of website
_Provide URLs and/or a description of where to look._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

---

## Reviewer tasks:

- [ ] Pulled branch and ran code locally (or viewed on preview deployment, if applicable)
- [ ] Reproduced the issue to review in the browser
- [ ] Reviewed code
- [ ] Updated Trello card (if applicable)

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[Clean up blog post tags.](https://trello.com/c/JPXzGji3/57-clean-up-blog-post-tags)

File: content/talks/css-rad.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ image:
   alt: Miriam Suzanne speaking at a conference
 slides: https://sliiides.netlify.com/css-is-rad/
 tags:
-  - Code
+  - Design
```
Comment (jgerigmeyer) [inline content/talks/css-rad.md]: @mirisuzanne Are the "code video" tags below in the events list being used for anything?

Comment (mirisuzanne) [reply content/talks/css-rad.md]: I don't think so. Probably a copy-paste hold-over…

File: content/_includes/page/list.njk [hunk 2]
```diff
@@ -3,18 +3,13 @@
...
-  } %}
-  {% set pages = event_pages[index.events] if index.events else 'mixed' %}
-  {% set list = collections[index.slug] if index.slug else none %}
-  {% set list = list | getEvents(pages) if index.events else list %}
-  {{ post.by_year(list, collections) }}
+  {% set list = collections[index] %}
+  {{ post.by_year(
+    list | removePage(page.url) if list else none,
+    collections
+  ) }}
```
Comment (jgerigmeyer) [inline content/_includes/page/list.njk]: I'm not 100% sure what this change does but 🤷‍♂ 👍

Comment (mirisuzanne) [reply content/_includes/page/list.njk]: Removes a chunk of code leftover from the way we used to handle events, and replaces it with a `removePage()` filter to keep e.g. `Jump Start Sass` (or any other resource that is also a tag index) from appearing in it's own resource list.

**Pull request merged**